homeassistant/helpers/config_validation.py

```diff
@@ -402,3 +402,9 @@ def validator(value):
     [vol.Any(SERVICE_SCHEMA, _SCRIPT_DELAY_SCHEMA, EVENT_SCHEMA,
              CONDITION_SCHEMA)],
 )
+
+
+def string_schema(*args: Sequence) -> Dict:
+    """Return a schema dict of Required values or Optional values."""
+    return {val if isinstance(val, vol.Optional) else vol.Required(val): string
+            for val in args}
```

Comment:
The default syntax we can work on: `tuples` is even more compact - Less is more :smile: 
```python
CC = 'c'
str_schema('a','b', (CC, '1'))
```

Usefulness is easy to show. The `str_schema` builder could remove 150-250 lines of code by the time we're done (since we don't have to import voluptuous for string only templates either)

On average 2.5x lines per file and there are still many voluptuous PRs outstanding.


```bash
# This function applies to 390 lines of code at this point in time

(hv) [pi@localhost components]$ grep --include "*.py" -R Required.*stri | wc -l
167
(hv) [pi@localhost components]$ grep --include "*.py" -R Optional.*stri | wc -l
220

#In 153 files
(hv) [pi@localhost components]$ grep -l --include "*.py" -R vol.*stri | wc -l
153
```